### DISCUSSION TITLE
How to batch convert ASCII to DFS2 on MIKE Zero Toolbox

### DISCUSSION THREAD

**Discussion by moinabyssinia** [DHI/mikeio#500] [Q&A]
I am certain that one can use the batch execution option on MIKE Zero toolbox to do this but I am new the command line interface of MIKE. Can you please give an example how to batch execute ASCII to DFS2? The manual (I checked even the 2023 version) doesn't give an example on that.

Thanks

**Comment by ecomodeller** [comment]
https://www.mikepoweredbydhi.com/support
